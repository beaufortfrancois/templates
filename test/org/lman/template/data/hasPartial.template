a-{{+partials.hasPartial_partial}}
b-{{+partials.hasPartial_partial}}
