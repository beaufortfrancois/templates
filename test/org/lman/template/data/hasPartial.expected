a-x1y2z
b-x1y2z
